list/harvesterhci.io.dashboard/index.vue

```diff
@@ -0,0 +1,532 @@
+<script>
+import dayjs from 'dayjs';
+import minMax from 'dayjs/plugin/minMax';
+import utc from 'dayjs/plugin/utc';
+import Loading from '@/components/Loading';
+import SortableTable from '@/components/SortableTable';
+import { allHash } from '@/utils/promise';
+import Poller from '@/utils/poller';
+import { parseSi, formatSi, exponentNeeded, UNITS } from '@/utils/units';
+import { REASON } from '@/config/table-headers';
+import {
+  EVENT, METRIC, NODE, HCI, SERVICE
+} from '@/config/types';
+import ResourceSummary from '@/components/ResourceSummary';
+import HardwareResourceGauge from '@/components/HardwareResourceGauge';
+import Tabbed from '@/components/Tabbed';
+import Tab from '@/components/Tabbed/Tab';
+import HarvesterMetrics from '@/components/HarvesterMetrics';
+import Upgrade from './Upgrade';
+
+dayjs.extend(utc);
+dayjs.extend(minMax);
+
+const PARSE_RULES = {
+  memory: {
+    format: {
+      addSuffix:        true,
+      firstSuffix:      'B',
+      increment:        1024,
+      maxExponent:      99,
+      maxPrecision:     2,
+      minExponent:      0,
+      startingExponent: 0,
+      suffix:           'iB',
+    }
+  }
+};
+
+const METRICS_POLL_RATE_MS = 20000;
```

Comment:
Can we use the mixins `metric-poller` to refactor the code?

https://github.com/harvester/dashboard/blob/fed0b43dc9c9041434eb0672fad66e1b581d096a/mixins/metric-poller.js#L6